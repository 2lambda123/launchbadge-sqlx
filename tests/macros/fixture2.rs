fn main() {}

mod test {
    #[sqlx::test(fixtures(path="../fixtures/postgres", scripts))]
    async fn should_fail(_pool: sqlx::PgPool) {}
}
