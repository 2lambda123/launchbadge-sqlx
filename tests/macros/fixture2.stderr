error: expected string literal
 --> tests/macros/fixture2.rs:4:27
  |
4 |     #[sqlx::test(fixtures(path="../fixtures/postgres", scripts))]
  |                           ^^^^
